x/bank/keeper/keeper.go

```diff
@@ -46,6 +46,9 @@ type Keeper interface {
 	DelegateCoins(ctx sdk.Context, delegatorAddr, moduleAccAddr sdk.AccAddress, amt sdk.Coins) error
 	UndelegateCoins(ctx sdk.Context, moduleAccAddr, delegatorAddr sdk.AccAddress, amt sdk.Coins) error
 
+	// GetAuthority Gets the address capable of executing governance proposal messages. Usually the gov module account.
+	GetAuthority() string
```

Comment:
why isn't this in the governance proposal content? Do we really need this feature if it can be modified via governance?

Reply:
This is the value that the `MsgSetSendEnabled.authority` field must equal. One side of the `==` is provided in the message, and this is the other side. If they're not equal, the `SetSendEnabled` endpoint returns an error.

This is intimately related to the stuff mentioned in https://github.com/cosmos/cosmos-sdk/pull/11981#discussion_r882923924.